bin/ch-run.c

```diff
@@ -208,6 +245,18 @@ int main(int argc, char *argv[])
       Tf (!args.c.writable || args.unsafe,
           "--write invalid when running by name");
       break;
+   case IMG_OCI_BUNDLE:
+      /* Parse config.json file -> containerize */
+      char *rootfs = path_join(args.c.img_ref, "rootfs");
+      char *conff = path_join(args.c.img_ref, "config.json");
+
+      Tf (path_exists(rootfs, NULL, true), "%s: No such file or directory", rootfs);
+      Tf (path_exists(conff, NULL, true), "%s: No such file or directory", conff);
```

Comment:
Check OCI bundle path provided contains ```config.json``` file and ```rootfs``` directory. Make sure they both exist